components/plans/create/AttributePicker.tsx

```diff
@@ -0,0 +1,231 @@
+
+  const addAttribute = (currentIndex: number, attribute: string) => {
+    let stopAttributes;
+
+    let stops = selectedAttributes;
+    stopAttributes = selectedAttributes[currentIndex];
+
+    if (!stopAttributes.includes(attribute)) stopAttributes.push(attribute);
+
+    stops[currentIndex] = stopAttributes;
+    setSelectedAttributes([...stops]);
+  };
+
+  const attributesSelected = () =>
+    selectedAttributes[0].length > 0 ? true : false;
+
+  const handleSuggestVenues = () => {
+    setSelectedTabIndex(2);
+    if (suggestions && suggestions?.length > 0) {
+      const firstSuggestion = suggestions[0];
+      if (map)
+        map.flyTo({
+          center: [
+            firstSuggestion.address.longitude,
+            firstSuggestion.address.latitude,
+          ],
+          zoom: 14,
+        });
+    }
+  };
+
+  return (
+    <div>
+      {attributes && (
+        <div>
+          <div>
+            <h2 className="text-md font-medium text-gray-700 mb-1">
+              Venue attributes
+            </h2>
+            <Picker
+              addAttribute={addAttribute}
+              attributes={attributes}
+              stopsIndex={attributesIndex}
+            />
+          </div>
+
+          <SelectedAttributes
+            attributes={selectedAttributes}
+            stopsIndex={attributesIndex}
+            removeStop={removeStop}
+            removeAttribute={removeAttribute}
+          />
+          <div className="w-full grid grid-cols-1 place-items-center">
+            <Button onClick={() => addStop(attributesIndex)} colour="gray">
+              <PlusIcon className="w-5" />
+            </Button>
+            {attributesSelected() && (
+              <div className="grid grid-cols-1 place-items-center pt-1">
+                <Button onClick={() => handleSuggestVenues()} colour="blue">
+                  <>
+                    View suggestions{" "}
+                    {suggestions ? (
+                      <>({suggestions.length})</>
+                    ) : (
+                      attributesSelected() && <>(0)</>
+                    )}
+                  </>
+                </Button>
+              </div>
+            )}
+          </div>
+        </div>
+      )}
+    </div>
+  );
+}
+
+function Picker({
+  addAttribute,
+  attributes,
+  stopsIndex,
+}: {
+  addAttribute: (index: number, attribute: string) => void;
+  attributes: string[];
+  stopsIndex: number;
+}) {
+  const [currentIndex, setCurrentIndex] = useState(0);
+  const pageSize = 8;
+  var pages = [];
+  for (var i = 0; i < attributes.length; i += pageSize) {
+    pages.push(attributes.slice(i, i + pageSize));
+  }
+
+  return (
+    <div>
+      <CardSlider currentIndex={currentIndex} setCurrentIndex={setCurrentIndex}>
+        {pages.map((page: string[], i: number) => (
+          <div key={i} className="grid grid-cols-4 grid-rows-2 w-full gap-2">
+            {page.map((attribute: string) => {
+              return (
+                <button
```

Comment:
consider creating a new component so buttons are consistent throughout. We could use the same one as before but create some variants?